aexpect/client.py

```diff
@@ -624,11 +626,10 @@ def _print_line(text):
                     return
                 try:
                     # See if there's any data to read from the pipe
-                    read_available_fds = select.select([tail_pipe], [], [],
-                                                       0.05)[0]
-                except (select.error, TypeError):
+                    read_available_fd = poller.poll(50)
```

Comment:
well `poller.poll` returns list of (fd, event), therefor `read_available_fd` is a misleading name. As we know we only registered tail_pipe, how about calling it `poll_status`?